infra.bs

```diff
@@ -1389,6 +1389,7 @@ its <a for=list>size</a> is zero.
 set of steps on each <a for=list>item</a> in order, use phrasing of the form
 "<a for=list>For each</a> |item| of <var ignore>list</var>", and then operate on |item| in the
 subsequent prose.
+The subsequent prose constitutes the |steps| of [=sequence iteration=] over the [=list=].
```

Comment:
I think this needs to tie to "sequence iteration" more directly. Right now the normative definition of For each is still roughly "performing a set of steps on each [item](https://whatpr.org/infra/451.html#list-item) in order".

Maybe

> To **iterate** over a list _list_ performing steps _steps_ on each item, perform sequence iteration given _list_ and _steps_. This can also be (and usually is) written as "For each _item_ of _list_: _steps_".

Edit: but see below, actually I think trying to share the code between lists and maps doesn't work well.